Demo/Demo.swift

```diff
@@ -42,6 +42,7 @@ public enum ComponentViews: String, CaseIterable {
     case refreshControl
     case horizontalSlide
     case newYearsView
+    case bottomSheetMechanis
```

Comment:
Small typo here, should be `bottomSheetMechanics`.

Reply:
Woops